### PR TITLE
LPS-183650 LPS-181500 | Refactor | Master

### PR DESCRIPTION
**Pull request by victorhcunha**
[LPS-183650](https://issues.liferay.com/browse/LPS-183650)
[LPS-181500](https://issues.liferay.com/browse/LPS-181500)

### CODE REVIEW

**Comment by victorhcunha** [comment]
ci:test:sf

**Comment by andreefarias** [comment]
Good job @victorhcunha! 

One question, there is some testcase that executes the navigate via the portlet menu? It's important to cover this part in at least one testcase.

**Comment by victorhcunha** [comment]
Hi @andreefarias !
There are many tests that keep using portlet step

**Comment by andreefarias** [comment]
Nice list @victorhcunha , thanks! Could replace all? Except for this testcase: `ObjectNotifications#CanCreateEmailTemplate`. This way the main test to create a template will follow the steps as the user. Does that make sense?

**Comment by victorhcunha** [comment]
Hi @andreefarias !
Make sense to me, now only two tests are still using the portlet menu to make those steps, CanSendEmailViaActionAdd (to Queue) and CanCreateEmailTemplate (to Template).

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase [hunk 1]
```diff
@@ -1221,10 +1179,7 @@ definition {
 		property portal.acceptance = "true";
 
 		task ("Given a Notification Template is created") {
-			ApplicationsMenu.gotoPortlet(
-				category = "Notifications",
-				panel = "Control Panel",
-				portlet = "Templates");
+			Notifications.notificationQueueAdmin();
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectActions.testcase:1182]: Use the: `Notifications.notificationTemplateAdmin();`

File: modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase [hunk 2]
```diff
@@ -118,10 +115,7 @@ definition {
 	@priority = 3
 	test CanDeleteAttachment {
 		task ("Given an object with an attachment field is created") {
-			ObjectAdmin.addObjectViaAPI(
-				labelName = "Custom Object 161928",
-				objectName = "CustomObject161928",
-				pluralLabelName = "Custom Objects 161928");
```
Comment (andreefarias) [inline modules/apps/object/object-test/src/testFunctional/tests/ObjectNotifications.testcase:124]: Don't replace the `addObjectViaAPI` to `Notifications.notificationTemplateAdmin();`

**Review by andreefarias — APPROVED** [review]
Nice @victorhcunha!
LGTM! 🎉

**Review by timpak — CHANGES_REQUESTED** [review]
Really great job with this refactor, @victorhcunha!! One small request: can you rename the macros to:
`goToNotificationQueueAdmin` 
`goToNotificationTemplateAdmin` 

The current macro names don't clearly explain what they're doing. Don't forget to sort the macros once you rename them!

**Comment by victorhcunha** [comment]
@timpak 
This macros are renamed by #764

**Comment by timpak** [comment]
@victorhcunha, you need to either include the macro rename in your changes. For example:
- `LPS-183650 Rename the notification navigation macros`
   - Use this commit to rename the macros
- `LPS-183650 Replace old navigation steps
   - Use this commit to apply the navigation macros to the tests

Instead of having someone else fix your changes like [this](https://github.com/liferay-bpm-qa/liferay-portal/pull/764/commits/7b7941291e5e2feafbd05a9c07eb12a3b11b5131), it's better to do it right the first time.

**Comment by victorhcunha** [comment]
ci:reopen

**Review by timpak — APPROVED** [review]
Great job re-organizing the commits @victorhcunha and @JerryAdrianoo!!

In the future, you can just use `Wordsmith` for commits like e30896c4df077224a2ef11312b9f99599ce76648. See https://github.com/liferay-bpm-qa/liferay-portal/pull/864/commits/985d542a9cfff734d1e17430a7333809cc686c4a for an example.

**Comment by timpak** [comment]
ci:forward

**Pull request closed without merging**